docs/quickstart.rst

```diff
@@ -61,11 +61,10 @@ ignored for now.)
 Preview your site
 -----------------
 
-Open a new terminal session and run the following commands to switch to your
-``output`` directory and launch Pelican's web server::
+Open a new terminal session and run the following commands to launch Pelican's
+web server::
 
-    cd ~/projects/yoursite/output
-    python -m pelican.server
+    pelican -l content
```

Comment:
Yes and no. `pelican content` will do something even if you have no setting file (default config will be enough). On the other hand, this makes a bunch of assumptions about the output path which may not be true. In that case it won't do anything. But... Given the context of this page, I guess it is fine. Just adding "run from project root" and removing the unnecessary `content` should suffice:

```
pelican --listen
```